audioapp/templates/audioapp/base.html

```diff
@@ -3,30 +3,33 @@
 <html>
 <head>
     {% tailwind_css %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
 </head>
-<body>
-    <header>
-        <nav>
-            {% if user.is_authenticated %}
-                <span>{{ user.username }}</span>
-            {% endif %}
-
-            <a href="{% url 'index' %}">Home</a>
-            {% if user.is_authenticated %}
-                <a href="{% url 'profile' %}">Profile</a>
-                <form method="post" action="{% url 'account_logout' %}">
-                    {% csrf_token %}
-                    <button type="submit">Sign Out</button>
-                </form>
-            {% else %}
-                <a href="{% url 'account_login' %}">Sign In</a>
-                <a href="{% url 'account_signup' %}">Sign Up</a>
-            {% endif %}
+<body class="bg-gray-100">
+    <header class="bg-white shadow">
+        <nav class="container mx-auto flex justify-between items-center p-4">
+            <div class="flex items-center space-x-4">
+                <a href="{% url 'for_you' %}" class="text-blue-500 hover:text-blue-700">For You</a>
+                <a href="{% url 'upload_audio' %}" class="text-blue-500 hover:text-blue-700">Upload Sound</a>
```

Comment:
It's typically ideal to duplicate any `hover:` styles for `focus:` states as well for keyboard users to ensure the indicators are consistent. You may see that these focus states can linger after clicking something, we've started using [focus-visible](https://developer.mozilla.org/en-US/docs/Web/CSS/:focus-visible) in TW as `focus-visible:` to still enable for keyboard users but have a softer effect for non-keyboard users.